scripts/install_KLU_Sundials.py

```diff
@@ -1,6 +1,6 @@
 # /// pyproject
 # [run]
-# requires-python = "<=3.9"
+# requires-python = "">=3.8, <=3.12""
```

Comment:
```suggestion
# requires-python = "">=3.8, <3.13""
```

Actually this should be a non-strict upper bound, since we support versions like Python 3.12.X as well

Reply:
> Actually this should be a non-strict upper bound, since we support versions like Python 3.12.X as well

I apologize for the oversight regarding the Python version range in our script. I've now corrected the version requirement to >=3.8, <3.13 as you suggested.